documentation/config_file_puppetserver.markdown

```diff
@@ -73,6 +73,19 @@ The `puppetserver.conf` file contains settings for Puppet Server software. For a
 * The `profiler` settings configure profiling:
 
     * `enabled`: If this is set to `true`, Puppet Server enables profiling for the Puppet Ruby code. The default is `false`.
+    
+* The `product` settings configure update checking and analytics data collection:
+
+    * `check_for_updates`: If set to `false`, Puppet Server will not automatically check for updates, and will not send analytics data to Puppet. 
```

Comment:
should this actually be `check-for-updates`? that's what https://github.com/puppetlabs/puppetserver/pull/1248/files seems to indicate at least.